integration/tests/epochs/recover_epoch/suite.go

```diff
@@ -12,10 +24,85 @@ type Suite struct {
 func (s *Suite) SetupTest() {
 	// use a shorter staking auction because we don't have staking operations in this case
 	s.StakingAuctionLen = 2
-	s.DKGPhaseLen = 50
-	s.EpochLen = 250
+	s.DKGPhaseLen = 10
```

Comment:
From our time woking on a way to trigger EFM, my understanding is that this short DKG phase is the thing triggering EFM. Assuming that's right, let's document that here.

Reply:
https://github.com/onflow/flow-go/pull/6156/commits/b64a5ca8512cc818af1fbfda224912b6d8bc84ad

Reply:
From [this comment](https://github.com/onflow/flow-go/pull/6156#discussion_r1671593729), it sounds like we're not sure what exactly is triggering EFM here. You mentioned that we will investigate further in a follow-up PR, which works for me.

In the meantime, I'd suggest adding a `TODO(EFM)` statement here and where we pause the collection node, noting that we aren't sure which of these two things is causing EFM, and this will be addressed in Pt. 2 of this issue.   (If there is a separate issue for Pt. 2, we can link to the issue number.)

Reply:
https://github.com/onflow/flow-go/pull/6156/commits/01bfe0d049bca22eee1c1b3fcbd520b466b38a8f